lib/jira/resource/watcher.rb

```diff
@@ -23,6 +23,13 @@ def self.all(client, options = {})
           issue.watchers.build(watcher)
         end
       end
+
+      def save!(user_id, path = nil)
+        path ||= new_record? ? url : patched_url
+        response = client.post(path, user_id.to_json)
```

Comment:
Adding a new watcher is always a post?
If that's the case that is why you override the method save! ?